src/components/icon/icon.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint global-require: "warn" */
```

Comment:
I'd suggest either 
* disabling/setting to warn level this rule for one single line in this file or
* setting it to warn level globally in `eslintrc`